src/mediapipe_internal/pythonnoderesource.hpp

```diff
@@ -21,7 +21,10 @@
 
 #if (PYTHON_DISABLE == 0)
 #include <pybind11/embed.h>  // everything needed for embedding
-
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_graph.h"
```

Comment:
You don't need to include in header, only in cpp. Here fwd declaration would be sufficient.